### ISSUE TITLE
Are created resources also watched?

### ISSUE THREAD

**Issue by raffaelespazzoli** [alauda/captain#5]
does this operator watch resources created by the helm charts? In other words if a chart created a deployment and some user changes the deployment spec, will this chart restored the desired version as defined by the chart?

**Comment by hangyan** [comment]
No, the operator only watches HelmRequest. I think helm itself also does not watch for resource changes
